src/collection-browser.ts

```diff
@@ -1981,5 +2000,19 @@ export class CollectionBrowser
     infinite-scroller.hidden {
       display: none;
     }
+
+    .sr-only {
```

Comment:
just noticed that we have a lot of `.sr-only` classes on most of the components, would it be possible to have a base `.sr-only` CSS template and reuse it by component if needed?

Reply:
Good idea! I'll create a shared importable template for that.

Reply:
All the `.sr-only` rules should now be drawing from a single shared template. Unfortunately, this entailed changing the indentation of some entire css blocks, which makes for a rather messy diff in a few places.

I think going forward, we should use the more flexible `static get styles() {}` form everywhere, instead of `static styles = ...`.